recipe/meta.yaml

```diff
@@ -1,40 +1,71 @@
-{% set version = "0.15rc1" %}
+{% set name = "movingpandas" %}
+{% set version = "0.17.2" %}
 
 package:
-  name: movingpandas
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/m/movingpandas/movingpandas-{{ version }}.tar.gz
-  sha256: 53b8eceaeec8a1c6f431874da6b63a3ca0ccca9a56c27ffd07c80a3b84194c1b
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a84bab2473a0ee254ac10bd562b4c4c955ea69820a033be0220abd4cd2476c6e
+  patches:
+    - patches/0001-fix-long-description-utf-8.patch  # [win]
 
 build:
   number: 0
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . -vv"
+  # fiona, rtree not available for s390x
+  skip: true  # [py<38 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
+  build:
+    - m2-patch    # [win]
   host:
-    - python >=3.7
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >=3.7
-    - fiona
+    - python
     - geopandas-base
+    # our geopandas-base includes already fiona
+    # see https://github.com/AnacondaRecipes/geopandas-feedstock/blob/db3ee7c59204c6ec5e797f6c208339e87ece8783/recipe/meta.yaml#L32
+    # - fiona
     - geopy
-    - matplotlib-base
+    - matplotlib-base  # [not win]
+    # win will require matplotlib (with tk/tcl)
+    - matplotlib       # [win]
```

Comment:
`matplotlib` goes with `pyqt`, see https://github.com/AnacondaRecipes/matplotlib-feedstock/blob/master/recipe/meta.yaml#L84. The final size of the user environment will become ~700Mb instead of 100Mb with `matplotlib-base`. TkAgg is a default backend for `matplotlib` (our matplotlib-base package too) https://matplotlib.org/stable/users/explain/figure/backends.html#the-builtin-backends.
Will it work with matplotlib-base?

Reply:
it will build with `matplotlib-base` but those tests that use `.plot` will fail on `win`. I restored the `matplotlib-base` for `win` now and all tests are enabled.

Reply:
strange, it works, thanks for the help!